security/WFCORE-5526_Remove_Legacy_Security_Realms.adoc

```diff
@@ -452,6 +452,90 @@ The following table identifies the tests in WildFly Core and WildFly affected by
 |Evolved https://issues.redhat.com/browse/WFLY-15057[WFLY-15057]
 |===
 
+|WildFly
```

Comment:
When I look in preview mode https://github.com/wildfly/wildfly-proposals/blob/c581b74477897429aff919444aa51675a5c8bc0f/security/WFCORE-5526_Remove_Legacy_Security_Realms.adoc This table is not formatted correctly.